api/scripts/wazuh-apid.py

```diff
@@ -119,10 +117,10 @@ def set_logging(log_path='logs/api.log', foreground_mode=False, debug_mode='info
         except ssl.SSLError:
             logger.error(str(APIError(2003, details='Private key does not match with the certificate')))
             sys.exit(1)
-        except IOError as e:
-            if e.errno == 22:
+        except IOError as exc:
+            if exc.errno == 22:
                 logger.error(str(APIError(2003, details='PEM phrase is not correct')))
```

Comment:
I think `str(...` should be changed with `f-strings`.

Reply:
No need to use str nor f-strings as APIError has __to_str__